src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin square($size) {
+  grid-template-columns: repeat($size, $catalog__card-width);
```

Comment:
Ensure that the variable `$catalog__card-width` is defined elsewhere in your SCSS files. If not, this will cause a compilation error when the mixin is used.